rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -72,14 +78,23 @@ exit 0
 %{gem_instdir}/locale
 %exclude %{gem_cache}
 %{gem_spec}
-%{foreman_bundlerd_dir}/%{gem_name}.rb
+%{foreman_bundlerd_plugin}
+%{foreman_apipie_cache_foreman}
```

Comment:
> The second argument to ln is NOT the output directory of plugin:apipie:cache:

The _first_ argument is the output directory of the rake task though - and the first argument is the target of the symlink.

The `ln -s` should create a symlink from /usr/share/foreman/public/apipie-cache/plugin/foreman_ansible to the directory containing the rake task output in the gems/foreman_ansible-1.2/ directory.

You then need `foreman_apipie_cache_foreman` listed to include the symlink, _and_ `foerman_apipie_cache_plugin` to include the rake output directory.